src/request/export/Export.js

```diff
@@ -24,7 +24,35 @@ class Export {
         this._exportWordsHandler = new ExportWords(request, this._wordsSucceeded.bind(this), this._reject.bind(this));
         this._exportFileHandler = new ExportFile(request, this._fileSucceeded.bind(this), this._reject.bind(this));
 
-        this._build();
+        /** @type {HTMLElement} */
+        const $moreExportOptionsPage = (
+            document.getElementById(Export.Pages.MORE_EXPORT_OPTIONS)
+                ? document.getElementById(Export.Pages.MORE_EXPORT_OPTIONS)
+                : this._buildMoreExportOptions()
+        );
```

Comment:
I feel this can be simplified to
```javascript
const $moreExportOptionsPage = document.getElementById(Export.Pages.MORE_EXPORT_OPTIONS) || this._buildMoreExportOptions();
```

Reply:
Same in `ExportFile.js` and `ExportWords.js`.